### PR TITLE
Some general updates/improvements.

### PR DESCRIPTION
**Pull request by oibind**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -13,38 +13,42 @@ plugins:
...
     - name: Twitter
-      url: "http://twitter.com/FarsetLabs"
+      url: "https://twitter.com/FarsetLabs"
+    - name: Slack
+      url: "https://slackin.farsetlabs.org.uk/"
+    - name: Blog
+      url: "https://blog.farsetlabs.org.uk/feed/"
     - name: Forum
-      url: "http://discourse.farsetlabs.org.uk/"
+      url: "https://discourse.farsetlabs.org.uk/"
```
Comment (andrewbolster) [inline _config.yml]: Discourse is dead but outside of scope for this PR

File: membership/index.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ Sponsored Memberships
 
 You may be eligible for sponsored membership if you're a member / employee / student of the following organisations:
 
-* Queen's University Belfast Students Union- [Apply here](http://www.qubsu.org/EnterpriseSU/Opportunities/FarsetLabs/)
+* Queen's University Belfast Students Union- [Apply here](https://www.qubsu.org/EnterpriseSU/Opportunities/FarsetLabs/)
```
Comment (andrewbolster) [inline membership/index.md]: qubsu don't support HTTPS

File: support/corporates.md [hunk 3]
```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
 
 ## Tibus Connect 
 
-<div style="float: right; padding: 0 20px 0 10px "><a href="http://www.tibus.com"><img src="{{site.base}}/assets/img/corp_logos/tibus.png"/></a></div>
+<div style="float: right; padding: 0 20px 0 10px "><a href="https://www.tibus.com"><img src="{{site.base}}/assets/img/corp_logos/tibus.png"/></a></div>
 
-[Tibus(Connect)](http://www.tibusconnect.com/) offer a full range of
+[Tibus(Connect)](https://www.tibusconnect.com/) offer a full range of
```
Comment (andrewbolster) [inline support/corporates.md]: should be https://www.tibus.com/

File: support/corporates.md [hunk 4]
```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
...
 connectivity solutions for business across the UK & Ireland.They are market
-partners of [Project Kelvin](http://www.tibusconnect.com/connect/project-
+partners of [Project Kelvin](https://www.tibusconnect.com/connect/project-
```
Comment (andrewbolster) [inline support/corporates.md]: https://www.tibus.com/

File: support/corporates.md [hunk 5]
```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
...
 kelvin/), and, along with [Hibernia
-Atlantic](http://www.hiberniaatlantic.com/), operate 102 network locations
+Atlantic](https://www.hiberniaatlantic.com/), operate 102 network locations
```
Comment (andrewbolster) [inline support/corporates.md]: now redirects to https://www.gtt.net/

File: support/corporates.md [hunk 6]
```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
...
 across North America and Europe, providing a multi-gigabit L2 connection that
 puts Northern Ireland business on to a level playing-field with the global
 internet marketplace.
 
 ## Weavers Court Business Park 
 
-<div style="float: right"><a href="http://www.weaverscourt.com"><img src="{{site.base}}/assets/img/corp_logos/weavers-court.png"/></a></div>
+<div style="float: right"><a href="https://www.weaverscourt.com"><img src="{{site.base}}/assets/img/corp_logos/weavers-court.png"/></a></div>
```
Comment (andrewbolster) [inline support/corporates.md]: doesn't support HTTPS

File: support/corporates.md [hunk 7]
```diff
@@ -26,22 +26,22 @@ If you are interested in becoming a Corporate Member or supporting one or more o
...
 
-Special thanks have to go to [Weavers Court Business Park](http://www.weaverscourt.com/); without the confidence and tireless support of Tom Ekin, Gary Cassels, and the whole Weavers Court team, Farset Labs simply couldn't be.
+Special thanks have to go to [Weavers Court Business Park](https://www.weaverscourt.com/); without the confidence and tireless support of Tom Ekin, Gary Cassels, and the whole Weavers Court team, Farset Labs simply couldn't be.
```
Comment (andrewbolster) [inline support/corporates.md]: doesn't support HTTPS

Comment (oibind) [reply support/corporates.md]: I'll resolve these all by the end of the day.

File: support/corporates.md (hunk 4 above)
Comment (oibind) [reply support/corporates.md]: This link is actually completely dead. I have removed it in a separate commit.

**Pull request merged**